LAD-AMA-Common/telegraf_utils/telegraf_config_handler.py

```diff
@@ -694,6 +694,13 @@ def handle_config(config_data, me_url, mdsd_url, is_lad):
 
     az_resource_id = data["compute"]["resourceId"]
 
+    # If the instance is VMSS then trim the last two values from the resource id ie - "/virtualMachines/0"
+    # Since ME expects the resource id in a particular format. For egs -
+    # IMDS returned ID - /subscriptions/<sub-id>/resourceGroups/<rg_name>/providers/Microsoft.Compute/virtualMachineScaleSets/<VMSSName>/virtualMachines/0
+    # ME expected ID- /subscriptions/<sub-id>/resourceGroups/<rg_name>/providers/Microsoft.Compute/virtualMachineScaleSets/<VMSSName>
+    if "virtualMachineScaleSets" in az_resource_id: 
```

Comment:
does this add / only in the beginning and not the end? az_resource_id = "/".join(az_resource_id.split("/")[:-2])
for e.g. /a/b/c

Reply:
yes it only adds it in the beginning